lms/static/js/student_account/views/FormView.js

```diff
@@ -172,7 +172,7 @@
 
                         if (key) {
                             test = this.validate(elements[i]);
-                            if (test.isValid) {
+                            if (test.isValid || $el.attr('id') === 'login-email') {
```

Comment:
I think you are misunderstanding here: the `id` attribute is for that element, so it means, in all cases, the email input element will not have error state if you do it like this. Either that or I'm misunderstanding what you are trying to really mean by id here.